test/infrastructure/docker/exp/internal/controllers/dockermachinepool_controller.go

```diff
@@ -63,7 +62,6 @@ const (
 // DockerMachinePoolReconciler reconciles a DockerMachinePool object.
 type DockerMachinePoolReconciler struct {
 	Client           client.Client
-	Scheme           *runtime.Scheme
```

Comment:
I see its not used in the Reconiler so removed it instead of passing mgr.GetScheme() from main.go.

Reply:
Makes sense, thx for checking!